### PR TITLE
Update CDD/HDD tool to use AvgRange component in stats panel

### PR DESCRIPTION
**Pull request by clhenrick**

### CODE REVIEW

**Comment by clhenrick** [comment]
@mukhtyar when I plug in the new `AvgRange` component similar to how its used in Annual Averages tool I get all `NaN` values:

image

File: src/routes/tools/degree-days/_ExploreData.svelte [hunk 1]
```diff
@@ -263,31 +265,42 @@
...
+          isHistorical="{true}"
+          groupList="{DEFAULT_STAT_GROUPS.filter((d) => !d.historical)}"
+          periodList="{DEFAULT_STAT_PERIODS.filter((d) => !d.historical)}"
+          periodId="mid-century"
           format="{formatFn}"
+          models="{$modelsStore}"
         />
       </li>
       <li class="block">
         <RangeAvg
```
Comment (mukhtyar) [inline src/routes/tools/degree-days/_ExploreData.svelte]: The new component name is `AvgRange`, I didn't want to break anything, so I used a different name.

Comment (clhenrick) [reply src/routes/tools/degree-days/_ExploreData.svelte]: Whoops, I forgot to update the component name and import.

**Comment by clhenrick** [comment]
[Tools: CDD/HDD](https://trello.com/c/CAGsPasI/50-tools-cdd-hdd)

**Comment by clhenrick** [comment]
@mukhtyar let me know if this looks good to you, thanks.

**Comment by mukhtyar** [comment]
@clhenrick looks good.

**Pull request merged**